finished-proposals.md

```diff
@@ -129,6 +129,6 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [static-class-features]: https://github.com/tc39/proposal-static-class-features
 [class-fields-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2021-04/apr-19.md#class-fields-private-methods-and-static-class-features-for-stage-4
 [regex-match-indices]: https://github.com/tc39/proposal-regexp-match-indices
-[regex-match-indices-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-11/nov-16.md#regexp-matches-indices-jsc-implementation-feedback
+[regex-match-indices-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-05/may-25.md#regexp-match-indices
 [await]: https://github.com/tc39/proposal-top-level-await
-[await-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2019-06/june-6.md#top-level-await-for-stage-3
+[await-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-05/may-25.md#top-level-await
```

Comment:
```suggestion
[await-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2021-05/may-25.md#top-level-await
```
we don't ever want to hardcode the default branch name, in case it changes in the future.